apps/gitness/src/pages-v2/repo/repo-summary.tsx

```diff
@@ -387,6 +317,10 @@ export default function RepoSummaryPage() {
         toRepoFiles={() => routes.toRepoFiles({ spaceId, repoId })}
         navigateToProfileKeys={() => (isMFE ? customUtils.navigateToUserProfile() : navigate(routes.toProfileKeys()))}
         isRepoEmpty={repository?.is_empty}
+        useRepoBranchesStore={useRepoBranchesStore}
+        renderProp={() => (
```

Comment:
this is a render prop, but it doesn't need to be _named_ "renderProp".
let's call it something like "branchSelectorRenderer" or something